samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -4,6 +4,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#if defined(__ZEPHYR__) && !defined(CONFIG_POSIX_API)
+
+#include <logging/log.h>
+#define LOG_LEVEL LOG_LEVEL_DBG
+LOG_MODULE_REGISTER(big_http_download);
```

Comment:
It's still unclear why this change is needed. It defines logger named "big_http_download". But it's not used by anything. This sample does all its output using printf().

Reply:
Should it be removed then?

Reply:
> Should it be removed then?

Unless you can explain why it's needed (i.e. give instructions how to reproduce some issue, which addition of this logger fixes), then yes, I don't see how it's useful.

Reply:
I thought Zephyrs logging API supports also POSIX. It looks like I was wrong. The logging module will be removed in next commit for sake of simplicity.